### PR TITLE
liblcf, easyrpg-player 0.7.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
The test needs updating.

File: Formula/liblcf.rb [hunk 1]
```diff
@@ -17,31 +16,32 @@ class Liblcf < Formula
...
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make"
-    system "make", "install"
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
```
Comment (carlocab) [inline Formula/liblcf.rb:30]:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```
Let's follow the template in `brew create --cmake`.

**Comment by carlocab** [comment]
```
In file included from src/main.cpp:18:
In file included from ./src/player.h:22:
./src/meta.h:26:10: fatal error: 'inireader.h' file not found
#include "inireader.h"
         ^~~~~~~~~~~~~
```
`easyrpg-player` probably doesn't support the new `libclf`.

**Comment by Ghabry** [comment]
liblcf has unit tests by now, so you could also use target "check" now for this.

Same applies to the Player

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**